test/ng/directive/ngOptionsSpec.js

```diff
@@ -1657,7 +1701,7 @@ describe('ngOptions', function() {
         scope.values.pop();
       });
 
-      expect(element.val()).toEqual('');
+      expect(element.val()).toEqualUnknownValue();
```

Comment:
Wish I had a better understanding of `unknownValue` usage in select + options. Had to revert the spec back to its form in ef894c87eaead76d90169113ab6acc9287654ea3.